pkg/transform/kubernetes/kubernetes.go

```diff
@@ -38,6 +38,13 @@ func createServices(app *spec.App) []runtime.Object {
 		if len(svc.Spec.Selector) == 0 {
 			svc.Spec.Selector = app.Labels
 		}
+		if s.Name == "" {
```

Comment:
It would be nice to make this into the function.
You already using it in two places here, and more will come.

Also if you have this in separate function you can unit test just that one function